ic-kit-runtime/src/canister.rs

```diff
@@ -1021,35 +1021,64 @@ impl Ic0CallHandlerProxy for Canister {
     }
 
     fn stable_size(&mut self) -> Result<i32, String> {
-        todo!()
+        Ok(self
+            .stable
+            .stable_size()
+            .try_into()
+            .unwrap_or(i32::max_value()))
```

Comment:
It should return an error if it can't fit in 32bit

Reply:
[2a2e6a8](https://github.com/qti3e/ic-kit/pull/1/commits/2a2e6a8219eaf85693376a065b17df6c1bbb480f)